test/fixtures.js

```diff
@@ -195,3 +195,76 @@ export const createColorList = (overrides = []) => [
   },
   ...overrides,
 ]
+export const createAnalysisConfig = (overrides = {}) => ({
+  label: 'Size',
+  analysis: 'size_of',
+  color: ['red', 'blue', 'white'],
+  range: [1, 2],
+  variable: 'sizeVariable',
+  category: 'measurements',
+  ...overrides,
+})
```

Comment:
IMO we want to use empty values for the fixture data where we can, and have the test files set the values to whatever they want. Otherwise we have specific data in these fixtures that might not make sense in some use cases, or might cause confusion.

```suggestion
export const createAnalysisConfig = (overrides = {}) => ({
  label: '',
  analysis: '',
  color: [],
  range: [],
  variable: '',
  category: '',
  ...overrides,
})
```

Reply:
[030d5fc](https://github.com/AMP-SCZ/dpdash/pull/344/commits/030d5fc1cf5b4f2b9a635bda08ca229effcb2bbb)

Reply:
[030d5fc](https://github.com/AMP-SCZ/dpdash/pull/344/commits/030d5fc1cf5b4f2b9a635bda08ca229effcb2bbb)